src/libtorch.cc

```diff
@@ -1576,9 +1568,7 @@ SetStringInputTensor(
                           " strings for inference input '" + name + "', got " +
                           std::to_string(element_idx))
                           .c_str()));
-    FillStringTensor(
-        input_list, tensor_offset + element_idx,
-        request_element_cnt - element_idx);
+    FillStringTensor(input_list, request_element_cnt - element_idx);
```

Comment:
Same here, only need to fill if `element_idx < request_element_cnt`, so add an additional check right before `FillStringTensor`

Reply:
Added a check for this